test/framework_tests/connect_dag.py

```diff
@@ -110,7 +110,7 @@ def run_test(self):
         self.assert_connection(eight_peers, 9, ConnectionType.DNS)
         self.assert_connection(nine_peers, 8, ConnectionType.IP)
 
-        self.log.info(f"Successfully ran the {os.path.basename(__file__)} scenario.")
+        self.log.info(f"Successfully ran the scenario file: {os.path.basename(__file__)} ")
```

Comment:
This outputs something a little funny:

```
2024-07-01T18:29:51+0000 - INFO - Successfully ran the scenario file: tmpama_so18.py
```

...because we send the scenario file to the server as a base64 string, then the server saves it as a temporary file, which is what is executed.

I'm ok with keeping this as long as it matches your intent.

Reply:
I saw that and think we should keep it. Having the actual file name may be helpful later with debugging.

Reply:
Actually, I'll reword it. If it comes across as a bit funny, I think it may be worth it to rework it a bit.